compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -128,6 +128,15 @@ if (cg->profiledPointersRequireRelocation() && secondChild->getOpCodeValue() ==
    }
 #endif
 
+   int64_t secondChildValue = is64bit ? secondChild->getLongInt() : secondChild->getInt();
```

Comment:
Before doing this you need to check if the second child is even a constant.  i.e., `secondChild->getOpCode().isLoadConst()`

Reply:
Condition added as required.